galaxykubeman/templates/job-predelete.yaml

```diff
@@ -27,4 +27,4 @@ spec:
         args:
           - "sh"
           - "-c"
-          - 'kubectl get deployment -n {{ .Release.Namespace }} -o name | grep -E "(web|job-[0-9]+|workflow)$" | xargs kubectl delete -n {{ .Release.Namespace }}; kubectl get pvc -n {{ .Release.Namespace }} -o name | grep -E "(cvmfs-gxy-data-pvc|cvmfs-gxy-main-pvc)$" | xargs kubectl delete -n {{ .Release.Namespace }}; kubectl get daemonset -n {{ .Release.Namespace }} -o name | grep "csi-cvmfsplugin" | xargs kubectl delete -n {{ .Release.Namespace }}; kubectl get pvc -n {{ .Release.Namespace }} -o name | grep -E "(cvmfs-alien-cache-pvc|galaxy-pvc)$" | xargs kubectl delete -n {{ .Release.Namespace }}'
+          - 'kubectl get job -n {{ .Release.Namespace }} -o name | grep -v "pre-delete" | xargs kubectl delete -n {{ .Release.Namespace }}; kubectl get deployment -n {{ .Release.Namespace }} -o name | grep -E "(web|job-[0-9]+|workflow|nginx)$" | xargs kubectl delete -n {{ .Release.Namespace }}; kubectl get pvc -n {{ .Release.Namespace }} -o name | grep -E "(cvmfs-gxy)$" | xargs kubectl delete -n {{ .Release.Namespace }}; kubectl get daemonset -n {{ .Release.Namespace }} -o name | grep "csi-cvmfsplugin" | xargs kubectl delete -n {{ .Release.Namespace }}; kubectl get pvc -n {{ .Release.Namespace }} -o name | grep -E "(cvmfs-alien-cache-pvc|galaxy-pvc)$" | xargs kubectl delete -n {{ .Release.Namespace }}'
```

Comment:
Fixes:
1. Don't delete the `pre-delete` job :)
2. Delete `-nginx` deployment as well
3. Added missing `-E` to `grep` command